tests/test_modules/test_contact.py

```diff
@@ -282,20 +300,93 @@ def test_contact_check_order(self, load_contact_objects_with_incorrect_order):
             contact_cls,
         ) = load_contact_objects_with_incorrect_order
 
-        from elastica.rod import RodBase
-        from elastica.rigidbody import RigidBodyBase
+        mock_rod = self.MockRod(2, 3, 4, 5)
+        mock_rigid_body = self.MockRigidBody(5.0, 5.0)
 
         with pytest.raises(TypeError) as excinfo:
             system_collection_with_contacts._finalize_contact()
-        assert "Systems provided to the contact class have incorrect order. \n"
-        " First system is {0} and second system is {1} . \n"
-        " If the first system is a rod, the second system can be a rod, rigid body or surface. \n"
-        " If the first system is a rigid body, the second system can be a rigid body or surface.".format(
-            RigidBodyBase, RodBase
-        ) in str(
-            excinfo.value
+        assert (
+            "Systems provided to the contact class have incorrect order. \n"
+            " First system is {0} and second system is {1}. \n"
+            " If the first system is a rod, the second system can be a rod, rigid body or surface. \n"
+            " If the first system is a rigid body, the second system can be a rigid body or surface."
+        ).format(mock_rigid_body.__class__, mock_rod.__class__) == str(excinfo.value)
```

Comment:
Why do you have this assert here? I thought it is checked in the contact function wrapper tests.

Reply:
This is for the NoContact class, the tests in contact function wrapper tests are for the other classes.